syncany-lib/src/main/java/org/syncany/operations/down/actions/FileCreatingFileSystemAction.java

```diff
@@ -70,20 +72,26 @@ protected void createFolder(FileVersion targetFileVersion) throws Exception {
 		}
 	}
 
-	protected void createFile(FileVersion reconstructedFileVersion) throws Exception {
-		File reconstructedFileInCache = assembleFileToCache(reconstructedFileVersion);		
-		moveFileToFinalLocation(reconstructedFileInCache, reconstructedFileVersion);	
+	protected void createFile(FileVersion reconstructedFileVersion) throws IOException {
+		File reconstructedFileInCache;
+		try {
+			reconstructedFileInCache = assembleFileToCache(reconstructedFileVersion);
```

Comment:
I generally dislike this pattern: 

```
X var;
try {
   X = ...;
}
catch (..) {
}
```

Couldn't we just move the `moveFileToFinalLocation` inside the `try`?